dabl-meta.yaml

```diff
@@ -1,6 +1,6 @@
 catalog:
     name: da-marketplace
-    version: 0.0.2
+    version: 0.0.2.1
```

Comment:
Not sure how to feel about this versioning scheme... not just because it adds a 4th component, but also it leaves the other bundled packages with mismatched versions (`0.0.2` vs `0.0.2.1` for just the adapter). 

I think I would prefer simply upping everything to `0.0.3` if a version change is needed

Reply:
Yep, I can get behind this.